easybuild/easyconfigs/p/Python/Python-2.7.9-goolf-1.7.20.eb

```diff
@@ -0,0 +1,119 @@
+name = 'Python'
+version = '2.7.9'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.9.1'
+scipyversion = '0.14.1'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+#    ('SQLite', '3.8.8.1'),
```

Comment:
why is this being excluded?

also, Tk should be included as a dep, see https://github.com/hpcugent/easybuild-easyconfigs/pull/1654